src/UI/Shared/WB.UI.Shared.Extensions/Services/MapUtilityService.cs

```diff
@@ -62,16 +85,28 @@ private static async Task<Basemap> GetLocalMap(IFileSystemAccessor fileSystemAcc
                         }
                         break;
                     }
+                    case ".shp":
+                    {
+                        var shapefileLayer = await GetShapefileAsFeatureLayer(existingMap.MapFullPath);
+                        if (shapefileLayer.LoadError == null)
+                        {
+                            return new Basemap(shapefileLayer);
```

Comment:
I don't get this code. Shapefile should be loaded on top of basemap that created either from file or web map as a feature layer. Loading is handled by GetShapefileAsFeatureLayer

Reply:
old code. removed